repo-projects/Directory.Build.targets

```diff
@@ -700,8 +694,7 @@
           Condition="'@(_InnerPackageCacheFiles)' != ''" />
   </Target>
 
-  <Target Name="MoveDotNetBuildLogs"
-        Condition="'$(IsUtilityProject)' != 'true'">
+  <Target Name="MoveDotNetBuildLogs" Condition="'$(IsUtilityProject)' != 'true'">
```

Comment:
What executes this target now?

Reply:
That target is still getting executed in the error and success cases:
- https://github.com/dotnet/dotnet/blob/f1a59fd29e766d6bdeb821f07b45287ecb92297a/repo-projects/Directory.Build.targets#L553
- https://github.com/dotnet/dotnet/blob/f1a59fd29e766d6bdeb821f07b45287ecb92297a/repo-projects/Directory.Build.targets#L837
- https://github.com/dotnet/dotnet/blob/f1a59fd29e766d6bdeb821f07b45287ecb92297a/repo-projects/Directory.Build.targets#L871
- https://github.com/dotnet/dotnet/blob/f1a59fd29e766d6bdeb821f07b45287ecb92297a/repo-projects/Directory.Build.targets#L876

Aside from the binlog that we control, repositories write to that directory and put additional logs there. The signing logs also land there.